lib/src/SessionManager.cc

```diff
@@ -40,23 +44,42 @@ SessionManager::SessionManager(trantor::EventLoop *loop, size_t timeout)
         }
         sessionMapPtr_ = std::unique_ptr<CacheMap<std::string, SessionPtr>>(
             new CacheMap<std::string, SessionPtr>(
-                loop_, 1.0, wheelNum, bucketNum));
+                loop_,
+                1.0,
+                wheelNum,
+                bucketNum,
+                [this](const std::string& key) {
+                    sessionEventsPtr_->onSessionStart(key);
+                },
+                [this](const std::string& key) {
+                    sessionEventsPtr_->onSessionDestroy(key);
+                }));
     }
     else if (timeout_ == 0)
     {
         sessionMapPtr_ = std::unique_ptr<CacheMap<std::string, SessionPtr>>(
-            new CacheMap<std::string, SessionPtr>(loop_, 0, 0, 0));
+            new CacheMap<std::string, SessionPtr>(
+                loop_,
+                0,
+                0,
+                0,
+                [this](const std::string& key) {
+                    sessionEventsPtr_->onSessionStart(key);
+                },
+                [this](const std::string& key) {
+                    sessionEventsPtr_->onSessionDestroy(key);
+                }));
```

Comment:
@an-tao Can you confirm 0 bucket/wheel works?

Reply:
> @an-tao Can you confirm 0 bucket/wheel works?

Yes, it works.